### PR TITLE
TeamG_Wiki_Entry

### PR DESCRIPTION
**Pull request by awadhutthube**
Hi, 
This is the MRSD Wiki Entry by TEAM G. Let me know if any changes are required. Thanks!

### CODE REVIEW

File: wiki/computing/aws-quickstart.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+## Using TMUX / SCREEN
+While using ssh for remote access, your connections may be terminated if there is no activity for a long period of time. While training large models, this may be a problem. As your connection is piped through ssh, once the machine is left idle, the connection breaks causing all running applications / processes to terminate. To avoid this we can use Tmux or Screen.
+
+TMUX​ is a ​terminal multiplexer​ for ​Unix-like​ ​operating systems​. It allows multiple ​terminal sessions to be accessed simultaneously in a single window. It is useful for running more than one ​command-line​ program at the same time. It can also be used to detach ​processes from their controlling terminals, allowing ​SSH​ sessions to remain active without being visible.
+*- Wikipedia*
+
+Here are a few important references in this regard
+1.  <https://www.hamvocke.com/blog/a-quick-and-easy-guide-to-tmux/>
+2.  <https://tmuxcheatsheet.com/>
```
Comment (smbryan) [inline wiki/computing/aws-quickstart.md:46]: These need to be in the `[title](url)` format. Jekyll has poor handling for HTML-esque formats.

File: wiki/computing/aws-quickstart.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+
+## Spot Instances
+
+A Spot Instance is an unused EC2 instance that is available for less than the On-Demand price. Because Spot Instances enable you to request unused EC2 instances at steep discounts, you can lower your Amazon EC2 costs significantly. The hourly price for a Spot Instance is called a Spot price. The Spot price of each instance type in each Availability Zone is set by Amazon EC2, and adjusted gradually based on the long-term supply of and demand for Spot Instances. Your Spot Instance runs whenever capacity is available and the maximum price per hour for your request exceeds the Spot price.
+
+Spot Instances are a cost-effective choice if you can be flexible about when your applications run and if your applications can be interrupted. For example, Spot Instances are well-suited for data analysis, batch jobs, background processing, and optional tasks. For more information, see Amazon EC2 Spot Instances - AWS Official
+
+More details at
+<https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-spot-instances.html#spot-get-started>
```
Comment (smbryan) [inline wiki/computing/aws-quickstart.md:65]: This too.

File: wiki/computing/aws-quickstart.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+## A word on Visual Studio Code:
+VsCode comes with a plugin for remotely logging into your machine. This way you can develop and edit your code using VS Code-
+The Visual Studio Code Remote - SSH extension allows you to open a remote folder on any remote machine, virtual machine, or container with a running SSH server and take full advantage of VS Code's feature set. Once connected to a server, you can interact with files and folders anywhere on the remote filesystem.
+No source code needs to be on your local machine to gain these benefits since the extension runs commands and other extensions directly on the remote machine.
+
+The following link provides details on the same.\
+<https://code.visualstudio.com/blogs/2019/07/25/remote-ssh>
```
Comment (smbryan) [inline wiki/computing/aws-quickstart.md:73]: And here.

**Comment by awadhutthube** [comment]
Hi, 
Thanks for the review. I have made the necessary changes. Let me know if there is any other issue.

**Comment by smbryan** [comment]
Looks good. Thank you!

**Pull request merged**